### PR TITLE
Added new metric cross_contamination_rate

### PR DESCRIPTION
**Pull request by justinjj24**
Added new metric to estimate cross_contamination_rate

### CODE REVIEW

**Comment by justinjj24** [comment]
- New metrics added to the pipeline
  - `cross_contamination_rate`
- Environment and dependencies for `verifybamid2` updated in the `containers/Dockerfile` and `containers/conda_env.yml` respectively
  - updated image pushed to dockerhub `cbig/npm-sample-qc:v0.9.0` 
- Added metric process in the `main.nf`
- Version `0.9.0` updated in the `nextflow.config`
- resource requirements for the metrics update in the `conf/base.config`
- Added resource data `1000g.phase3.100k.b38.vcf.gz.dat` in to `resources` folder
- Computing the metrics in the `bin/metrics/metrics.py` and `/bin/compile_metrics.py` updated accordinly
- Sample `NA12878` used in the unittest `(s3://1000genomes-dragen-3.7.6/data/individuals/hg38-graph-based/NA12878/NA12878.bam)`
- Unittest environment local `(Ubuntu 22.04)` and `nextflow tower`

File: bin/metrics/metrics.py
Comment (justinjj24) [inline bin/metrics/metrics.py:1]: - computing of cross_contamination_rate included
- update of np.float64 due to numpy update

File: conf/base.config
Comment (justinjj24) [inline conf/base.config:1]: - added resource for `verifybamid2` process

File: conf/resources.config
Comment (justinjj24) [inline conf/resources.config:1]: - added verifybamid2 resource data
- added ACMG3.0 73 genes bed file

File: containers/Dockerfile
Comment (justinjj24) [inline containers/Dockerfile:1]: - Added verifybamid env to create new image

File: containers/conda_env.yml
Comment (justinjj24) [inline containers/conda_env.yml:1]: - Added dependencies for verifybamid2

File: main.nf
Comment (justinjj24) [inline main.nf:1]: - Added `cross_contamination_rate` process and included in the workflow

File: conf/resources.config
Comment (nicolas-bertin) [reply conf/resources.config:1]: might be better to have `1000g.phase3.100k.b38.vcf.gz.dat.*  in `/resources/`verifybamid/ rather than `/resources/`

**Review by mhebrard — COMMENTED** [review]
I am a bit confuse by the value of cross contamination. FREEMIX gives "1.8e-6". The workflow report 1.8e-4... Should we then name this metric "percent_cross_contamination". reason is we do not report the "rate" as calculated by the tool but the "%"

File: conf/resources.config [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 params {
     reference = "s3://1000genomes-dragen-3.7.6/references/fasta/hg38.fa"
+    vbi2_svdprefix = "1000g.phase3.100k.b38.vcf.gz.dat"
```
Comment (mhebrard) [inline conf/resources.config:8]: What is this param used for ?

**Comment by justinjj24** [comment]
As discussed the latest commit [be8e6bc](https://github.com/c-BIG/NPM-sample-qc/pull/90/commits/be8e6bc8d75685bb11cd5576af8816f1d195a223) to keep the metric name `cross_contamination_rate` and present the actual value of `FREEMIX` without converting to percentage.

**Review by mhebrard — APPROVED** [review]

**Pull request merged**